src/GraphQL/Types/ISchema.cs

```diff
@@ -5,48 +5,138 @@
 
 namespace GraphQL.Types
 {
+    /// <summary>
+    /// The schema for the GraphQL request. Contains references to the 'query', 'mutation', and 'subscription' base graph types.<br/>
+    /// <br/>
+    /// Also allows for adding custom directives, additional graph types, and custom value converters.<br/>
+    /// <br/>
+    /// <see cref="Schema"/> only requires the <see cref="Schema.Query">Query</see> property to be set; although commonly the <see cref="Schema.Mutation">Mutation</see> and/or <see cref="Schema.Subscription">Subscription</see> properties are also set.
+    /// </summary>
     public interface ISchema
     {
+        /// <summary>
+        /// Returns true once the schema has been initialized
+        /// </summary>
         bool Initialized { get; }
 
+        /// <summary>
+        /// Initializes the schema. Called by <see cref="IDocumentExecuter"/> before validating or executing the request.<br/>
+        /// <br/>
+        /// Note that middleware cannot be applied once the schema has been initialized. See <see cref="ExecutionOptions.FieldMiddleware"/>.
+        /// </summary>
         void Initialize();
 
+        /// <summary>
+        /// The <see cref="IFieldNameConverter"/> used by the schema. This is set by <see cref="IDocumentExecuter"/> to the converter passed to it within <see cref="ExecutionOptions.FieldNameConverter"/>.
+        /// </summary>
         IFieldNameConverter FieldNameConverter { get; set; }
 
+        /// <summary>
+        /// The 'query' base graph type; required
+        /// </summary>
         IObjectGraphType Query { get; set; }
 
+        /// <summary>
+        /// The 'mutation' base graph type; optional
+        /// </summary>
         IObjectGraphType Mutation { get; set; }
 
+        /// <summary>
+        /// The 'subscription' base graph type; optional
+        /// </summary>
         IObjectGraphType Subscription { get; set; }
 
+        /// <summary>
+        /// Returns a list of directives supported by the schema.<br/>
+        /// <br/>
+        /// Directives are used by the GraphQL runtime as a way of modifying execution
+        /// behavior. Type system creators do not usually create them directly.<br/>
+        /// <br/>
+        /// <see cref="Schema"/> intializes the list to include <see cref="DirectiveGraphType.Include"/>, <see cref="DirectiveGraphType.Skip"/> and <see cref="DirectiveGraphType.Deprecated"/> by default.
+        /// </summary>
         IEnumerable<DirectiveGraphType> Directives { get; set; }
 
+        /// <summary>
+        /// Returns a list of all the graph types utilized by this schema
```

Comment:
In fact, the schema contains many unnecessary scalars. I have PL to remove them. Let the comment remain so.